pkg/networkmanager/interfaces.js

```diff
@@ -2597,6 +2615,17 @@ PageNetworkInterface.prototype = {
                 }
             }
 
+            function render_zone_row() {
+                var rows = [ ];
+
+                function add_row(fmt, args) {
+                    rows.push(cockpit.format(fmt, args));
+                }
+
+		add_row(zone_text(settings.connection.zone), settings.connection.zone);
```

Comment:
The first argument of `add_row` is a format string for `cockpit.format`, but you pass the zone name directly.